contrib/babelfishpg_tsql/src/pltsql_function_probin_handler.c

```diff
@@ -434,6 +441,12 @@ buildTypmodArray(CreateFunctionStmt *stmt, int **typmod_array_p, int *array_len_
 				else
 				{
 					(*typmod_array_p)[i] = ptr->val.ival.ival;
+				/**
+			 	* For TSQL procedures and functions, MAX typmod for sys.(N)(VAR)CHAR, sys.VARBINARY datatypes
```

Comment:
Can we add comment what dataypes previous call will cover and what this will cover? Instead of having genric comment.

Reply:
Had added the specific datatypes and typmod typ (max/default) along with proc/func specification. 
Updated the comment as per our discussion.